Easydict/NewApp/Feature/Shortcut/Shortcut+Default.swift

```diff
@@ -12,6 +12,12 @@ import Magnet
 extension Shortcut {
     // set defalut for app shortcut
     func setDefaultForAppShortcut() {
+        Defaults[.selectionShortcut] = KeyCombo(key: .d, cocoaModifiers: .option)
```

Comment:
Creating a new function named `setDefaultForGlobalShortcut` will be more clearly

Reply:
Resolved with 27bca4f